src/pages/AddItem/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import Button from '../../components/button';
+import Title from '../../components/title';
+import ContentContainer from '../../components/content-container';
+import AddForm from '../../components/add-form';
+import { addProduct } from '../../utils/firebaseUtils';
+import './styles.css';
+
+export const AddItemPage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [message, setMessage] = useState(null);
+  const [error, setError] = useState(false);
+  const [formState, setFormState] = useState({
+    productName: '',
+    timeFrame: '7',
+    lastPurchaseDate: null,
+  });
```

Comment:
This is the first time I see a useState llike this one :eyes: I think it is a great approach though.

What if you create a custom hook for forms?

Reply:
It's the first time I see it too! This is a great job done by @susird! 🥳 

About the custom hook, we didn't get that idea. In what ways do you think it would help our functionality? I understand little about custom hooks 🤔